### PR TITLE
Symfony to beta

### PR DESCRIPTION
**Pull request by wadjei**
@brettmc - regarding https://github.com/opentelemetry-php/contrib-sdk-bundle/releases/tag/0.0.18

I managed to get the package to work with the beta release in my particular case by removing references to Jaeger (in favour of the closely compatible Zipkin) and tweaking the configuration loader for Exporters.

I know it's a bit of a hack and probably needs the configuration to be redefined to be compatible with the underlying changes, but was this the only thing blocking compatibility with the beta release or are there other issues?

Does the alternative StreamTransport also need to be supported or is hardcoding the package to the PsrTransport as I've done here sufficient?

### CODE REVIEW

**Comment by brettmc** [comment]
> I know it's a bit of a hack and probably needs the configuration to be redefined to be compatible with the underlying changes, but was this the only thing blocking compatibility with the beta release or are there other issues?

Great question! I think the best guide is whether you can get it to run and emit telemetry without errors, then I think we should commit it so that others can test it out.

> Does the alternative StreamTransport also need to be supported or is hardcoding the package to the PsrTransport as I've done here sufficient?

Yes, other transports should be supported (especially gRPC), but if you have trouble with it you can just add a `todo` and we will create an issue for follow-up.

**Comment by wadjei** [comment]
> > I know it's a bit of a hack and probably needs the configuration to be redefined to be compatible with the underlying changes, but was this the only thing blocking compatibility with the beta release or are there other issues?
> 
> Great question! I think the best guide is whether you can get it to run and emit telemetry without errors, then I think we should commit it so that others can test it out.
> 
> > Does the alternative StreamTransport also need to be supported or is hardcoding the package to the PsrTransport as I've done here sufficient?
> 
> Yes, other transports should be supported (especially gRPC), but if you have trouble with it you can just add a `todo` and we will create an issue for follow-up.

Great! The fork I currently have does emit telemetry (which made me very happy!) - if the other transports support instantiation with the static `discover()` method then I suppose the class could be added as an optional configuration parameter.

I'll have a play with this in due course and see if I can get a nicer PR to cover these cases

**Comment by wadjei** [comment]
just posted a comment on the upstream PR after thinking it was this one and that it had been automatically closed (https://github.com/open-telemetry/opentelemetry-php-contrib/pull/58)...

TL;DR - I plan to refine this but have other commitments for most of this week
